tools/travis/runTests.sh

```diff
@@ -21,14 +21,14 @@ set -e
 # Build script for Travis-CI.
 
 SECONDS=0
-SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+SCRIPTDIR=$(cd "$(dirname "$0")" && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
 
 cd $ROOTDIR
 cat whisk.properties
 TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage
 # disabled test until https://github.com/apache/incubator-openwhisk/issues/4169 is resolved
-TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage # :tests:testSwaggerCodegen
+#TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage # :tests:testSwaggerCodegen
```

Comment:
Good catch.  I meant to leave this line as 
`#TERM=dumb ./gradlew :tests:testCoverageLean :tests:reportCoverage :tests:testSwaggerCodegen`
I'm not sure how the comment ended up in the wrong place.